packages/core-data/src/hooks/index.ts

```diff
@@ -0,0 +1,2 @@
+export { useEntityRecord } from './use-entity-record';
```

Comment:
This part is a bit confusing since there is already the following in `src/index.js`:
https://github.com/WordPress/gutenberg/blob/4f108af9b6f7ee405be53a8364c386c59254b0a4/packages/core-data/src/index.js#L71-L78

Should it be inside this file and re-exported only from `src/index.js`?

Reply:
Yeah, this file is just there to make the docgen happy. You are right it would be more readable to reference this file in `src/index.js` – let's do that.